src/tkn.ts

```diff
@@ -580,7 +587,7 @@ export class TknImpl implements Tkn {
             const tknDownMsg = `The current user doesn't have the privileges to interact with tekton resources.`;
             return [new TektonNodeImpl(null, tknDownMsg, ContextType.TKN_DOWN, TknImpl.instance, TreeItemCollapsibleState.None)];
         }
-        if (result.stderr.indexOf(`the server doesn't have a resource type "pipeline"`) > -1) {
+        if (getStderrString(result.error).indexOf(`the server doesn't have a resource type "pipeline"`) > -1) {
```

Comment:
I got `Cannot read property 'indexOf' of undefined` there when doing test, you need to check that `getStderrString ` result is not `undefined` before you do `indexOf ` check.
I think you can modify that if by adding `result.error &&` first.